libs/EXTERNAL/libezgl/include/ezgl/application.hpp

```diff
@@ -408,6 +422,18 @@ class application {
  * @param new_setting The new state of disable_event_loop flag
  */
 void set_disable_event_loop(bool new_setting);
+#endif
 }
 
+#ifdef ECE297
```

Comment:
The main difference between the two code paths is that the default one defines the set_disable_event_loop() function inside the ezgl namespace which is cleaner. For auto-marking, the compiler errors out since the ezgl namespace is defined in the students' code which we link with later. I'll try to find a way to modify the auto-marker.

I'll expand the comment as suggested.